apps/fipsinstall.c

```diff
@@ -431,7 +431,7 @@ int fipsinstall_main(int argc, char **argv)
     if (read_buffer == NULL)
         goto end;
 
-    mac = EVP_MAC_fetch(NULL, mac_name, NULL);
```

Comment:
Not sure this one is useful..

Reply:
I don't think it changes anything either but it brings all uses into consistency.  We never know where the copy/paste source will be....